internal/clients/team4/forage.go

```diff
@@ -0,0 +1,23 @@
+package team4
+
+import (
+	"math"
+	"math/rand"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+)
+
+// DecideForage makes a foraging decision, currently you can only forage deer, but fishing will be available later
+// the forageContribution can not be larger than the total resources available
+func (c *client) DecideForage() (shared.ForageDecision, error) {
+	ft := int(math.Round(rand.Float64())) // 0 or 1 with equal prob.
+	return shared.ForageDecision{
+		Type:         shared.ForageType(ft),
+		Contribution: shared.Resources(rand.Float64() * 20),
```

Comment:
maybe there should be additional guard to contribute not more than say 20% of our resources?
or if we're poor af we choose the less expensive option?

Reply:
Oh this will be overwritten by Toby, these are just the default function implementations done in baseclient